docs/rhoas/rhoas-produce-consume/quickstart.yml

```diff
@@ -5,6 +5,14 @@ metadata:
   annotations:
     draft: false
     order: 3
+  labels:
+    service-category/application-services: true
+    product/rhosak: true
+    product/rhosr: true    
+    product/rhoad: true
+    product/rhoam: true
+    product/rhoc: true
```

Comment:
Although this guide uses the CLI, I'm not sure you'd say that it applies to all services. It's fairly specific to producing and consuming messages using a RHOSAK instance.

Reply:
Agreed -- rhosak only for the current text.

Reply:
I'm definitely not as familiar with these as you all are so I trust your changes. :)

Reply:
removed ref to 
  product/rhosr: "true"    
    product/rhoad: "true"
    product/rhoam: "true"
    product/rhoc: "true"